croncat/src/config/mod.rs

```diff
@@ -18,6 +18,7 @@ pub struct ChainConfig {
     pub contract_address: String,
     pub gas_prices: f64,
     pub gas_adjustment: f64,
+    pub polling_duration_secs:u64,
```

Comment:
Could you follow the instructions in the README on how to setup the git hooks so we don't push unformatted code.